python/test/backend/extension_backend.c

```diff
@@ -0,0 +1,42 @@
+#include <Python.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+static PyObject *getDeviceProperties(PyObject *self, PyObject *args) {
+  // create a struct to hold device properties
+  return Py_BuildValue("{s:i, s:i, s:i, s:i, s:i}", "max_shared_mem", 1024,
+                       "multiprocessor_count", 16, "sm_clock_rate", 2100,
+                       "mem_clock_rate", 2300, "mem_bus_width", 2400);
+}
+
+static PyObject *loadBinary(PyObject *self, PyObject *args) {
+  // get allocated registers and spilled registers from the function
+  int n_regs = 0;
+  int n_spills = 0;
+  int mod = 0;
+  int fun = 0;
+  return Py_BuildValue("(KKii)", (uint64_t)mod, (uint64_t)fun, n_regs,
```

Comment:
ditto

Reply:
It is a dummy value and is only for test purposes. In a real case, it should come from the function info.